src/microbe_stage/editor/CellEditorComponent.cs

```diff
@@ -1480,6 +1480,9 @@ private bool IsValidPlacement(OrganelleTemplate organelle)
             (organelle.Definition.RequiresNucleus && !HasNucleus))
             return null;
 
+        if (organelle.Definition.Unique)
+            OnOrganelleToPlaceSelected(null!);
```

Comment:
Overriding nullability for this should not be done like this. Maybe you should write a new method called "DeselectOrganelleToPlace"?

Reply:
Done.